dataline-scheduler/src/main/java/io/dataline/scheduler/WorkerRunner.java

```diff
@@ -89,7 +89,11 @@ public void run() {
         new WorkerRun<>(
                 jobId,
                 syncInput,
-                new SingerSyncWorker(
+                // todo (cgardens) - still locked into only using SingerTaps and Targets. Next step
```

Comment:
I see nothing related to singer in that code. I think this comment is mis-leading

Reply:
i'll fix this. going to dependency inject the factories and then this comment will make sense again.